docs/bigquery/snippets.py

```diff
@@ -85,49 +85,79 @@ def delete(self):
         self._wrapped.close()
 
 
-def test_client_list_datasets(client):
+def test_list_datasets(client):
     """List datasets for a project."""
+    # [START bigquery_list_datasets]
+    # client = bigquery.Client()
+    datasets = list(client.list_datasets())
+    project = client.project
 
-    def do_something_with(_):
-        pass
-
-    # [START client_list_datasets]
-    for dataset in client.list_datasets():  # API request(s)
-        do_something_with(dataset)
-    # [END client_list_datasets]
+    if datasets:
+        print('Datasets in project {}:'.format(project))
+        for dataset in datasets:  # API request(s)
+            print('\t{}'.format(dataset.dataset_id))
+    else:
+        print('{} project does not contain any datasets.'.format(project))
+    # [END bigquery_list_datasets]
 
 
 def test_create_dataset(client, to_delete):
     """Create a dataset."""
-    DATASET_ID = 'create_dataset_{}'.format(_millis())
+    dataset_id = 'create_dataset_{}'.format(_millis())
 
-    # [START create_dataset]
-    # DATASET_ID = 'dataset_ids_are_strings'
-    dataset_ref = client.dataset(DATASET_ID)
+    # [START bigquery_create_dataset]
+    # client = bigquery.Client()
+    # dataset_id = 'my_dataset'  # must be unique within your project
```

Comment:
Is the "must be unique" comment necessary? If we do want to indicate this, I would prefer we add a comment to the `create_dataset()` line that is could throw [google.api_core.exceptions.AlreadyExists](https://googlecloudplatform.github.io/google-cloud-python/latest/core/exceptions.html#google.api_core.exceptions.AlreadyExists).

Reply:
I see you added the comment. Thanks!